CMakeLists.txt

```diff
@@ -38,7 +38,38 @@ set_target_properties(
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
-find_package(ICU COMPONENTS uc i18n REQUIRED)
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+  message(STATUS "Apple system detected.")
+  # People who run macOS often use brew.
+  if(EXISTS /opt/homebrew/opt/icu4c)
+    message(STATUS "icu is provided by homebrew.")
+    ## This is a bit awkward, but it is a lot better than asking the
+    ## user to figure that out.
+    list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/icu4c/include")
+    list(APPEND CMAKE_LIBRARY_PATH "/opt/homebrew/opt/icu4c/lib")
+  endif()
+endif()
+
+## If we put 'REQUIRED' under find_package, it is a hard error and
+## we get no chance to provide instructions to the user.
+find_package(ICU COMPONENTS uc i18n)
+
+### If the user does not have ICU, let us help them with instructions:
+if (NOT ICU_FOUND)
+    if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+      if(EXISTS /opt/homebrew)
```

Comment:
Intel based macOS has a different path for homebrew.

Reply:
Fixed.